web/modules/custom/rdf_entity/src/RdfPermissions.php

```diff
@@ -41,19 +41,25 @@ public function rdfTypePermissions() {
    */
   protected function buildPermissions(RdfEntityType $type) {
     $type_id = $type->id();
-    $type_params = array('%type_name' => $type->label());
+    $type_params = ['%type_name' => $type->label()];
 
-    return array(
-      "create $type_id rdf entity" => array(
+    return [
+      "create $type_id rdf entity" => [
```

Comment:
There is no standard pattern for permissions in Drupal. They are human readable. As far as human readability goes, I much prefer "create owner rdf entity" over "create owner rdf_entity entity", especially because this is in the contributed RDF Entity module.